chautari-rental-service.yml

```diff
@@ -0,0 +1,938 @@
+        '200':
+          description: "successful operation"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResourceResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        '401':
+          description: 'Unauthorized, Invalid or Missing Access Token'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        '403':
+          description: 'Access Denied, Insufficient Scope privilege Or Expired Access Token'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        '404':
+          description: Resource Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        '429':
+          description: Too Many Requests
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        default:
+          description: Other Errors
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+components:
+  securitySchemes:
+    basicAuth:
+      type: http
+      scheme: basic
+  schemas:
+    ErrorModel:
+      type: object
+      properties:
+        developerMessage:
+          type: string
+          description: >-
+            Verbose, plain language description of the problem for the app
+            developer with hints about how to fix it.
+        userMessage:
+          type: string
+          description: Pass this message on to the app user if needed
+        errorCode:
+          type: string
+          description: Error code defined for the error
+        moreInfo:
+          type: string
+          description: Any additional information on the error (e.g. website links)
+    Credential:
+      type: object
+      required:
+        - userName
+        - password
+      properties:
+        userName:
+          type: string
+        password:
+          type: string
+        security-questions:
+          type: array
+          items:
+            $ref: '#/components/schemas/SecurityQuestionList'
+    SecurityQuestionList:
+      type: object
+      properties:
+        question:
+          type: string
+        answer:
+          type: string
+    AuthenticationResponse:
+      type: object
+      properties:
+        message:
+          type: string
+    UserAccountDetail:
+      type: object
+      properties:
+        user-type:
```

Comment:
we dont this user type while sign off, we can add something called request type when user intersecting with app